testing/test_terminal.py

```diff
@@ -668,6 +668,26 @@ def test_showlocals():
             ]
         )
 
+    def test_showlocals_short(self, testdir):
+        p1 = testdir.makepyfile(
+            """
+            def test_showlocals_short():
+                x = 3
+                y = "x" * 5000
+                assert 0
+        """
+        )
+        result = testdir.runpytest(p1, "-l", "--tb=short")
+        result.stdout.fnmatch_lines(
+            [
+                "test_showlocals_short.py:*",
+                "    assert 0",
+                "E   assert 0",
+                "        x* = 3",
+                "        y* = 'xxxxxx*",
```

Comment:
The `"*"` is not needed with `"x*"` and `"y*"`, is it?  (I think it is better to have explicit matching here)
Also with regard to the `"x" * 5000` - I would expect this to be not truncated (at least in verbose mode), so you could add it there via string formatting.

Reply:
Replaced with explicit test. The output here is truncated since there's no `-vv` there, but probably truncation is orthogonal to the change here.